Tests/OutlineViewTests/TreeMapTests.swift

```diff
@@ -0,0 +1,139 @@
+import XCTest
+@testable import OutlineView
+
+final class TreeMapTests: XCTestCase {
+    struct TestItem: Identifiable, Equatable {
+        var id: Int
+        var children: [TestItem]?
+    }
+
+    /// A basic TreeMap with 5 root objects (1 through 5),
+    /// where all but item 5 are internal nodes, and all
+    /// are collapsed.
+    static var TestTree: TreeMap<Int> {
```

Comment:
Variable names should start with a lower case letter. 

```suggestion
    static var testTree: TreeMap<Int> {
```

Optional (entirely your choice): You can make it private and turn it into an instance variable so that you don't need to add `Self.`.

```suggestion
    private var testTree: TreeMap<Int> {
```

Reply:
Fixed in https://github.com/Sameesunkaria/OutlineView/pull/10/commits/b00ac09c8a614dcb7e0146a63f3a8fb55c39dbf8 and https://github.com/Sameesunkaria/OutlineView/pull/10/commits/3221fc4d714047e4f0ac267c9155dffb2a8bcbac